scripts/check-bundled-ca-certs-expirations.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Script which errors out if any of the bundled certs will expire in 24 months or sooner.
+"""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+if False:
+    from typing import List
```

Comment:
If this is never True, do we need to include it?

Reply:
It's only used for type checking purposes is only evaluated when running mypy.

We can drop "if False" part once we drop support for Python 2.